gctscript/vm/vm_test.go

```diff
@@ -25,7 +25,6 @@ var (
 	testBrokenScript         = filepath.Join("..", "..", "testdata", "gctscript", "broken.gct")
 	testScriptRunner         = filepath.Join("..", "..", "testdata", "gctscript", "timer.gct")
 	testScriptRunner1s       = filepath.Join("..", "..", "testdata", "gctscript", "1s_timer.gct")
-	testScriptRunnerInvalid  = filepath.Join("..", "..", "testdata", "gctscript", "invalid_timer.gct")
```

Comment:
Why was this removed?

Reply:
It was not being used.

Reply:
The file is still there. I think we should re-add it, or remove the file if its a broken test